Doc/library/ssl.rst

```diff
@@ -764,6 +769,15 @@ Constants
 
    .. versionadded:: 3.4
 
+.. data:: OP_NO_TLSv1_3
+
+   Prevents a TLSv1.3 connection. This option is only applicable in conjunction
+   with :const:`PROTOCOL_TLS`. It prevents the peers from choosing TLSv1.3 as
+   the protocol version. TLS 1.3 is available with OpenSSL 1.1.1 or later.
+   With older versions, the flag defaults to *0*.
```

Comment:
@pitrou: note that in this specific case, the flag being zero means it exactly implements the standard semantics (it successfully disables the feature, because the feature is always disabled). That's not the case for most other flags.

I don't really care that much; obviously the other way has worked fine too. But @tiran's approach has some mild but nice benefits and I'm not sure why this is a point of contention at all – AFAICT all your arguments have been that it doesn't matter either way; I haven't seen a single argument for why having these flags always available is harmful?